cluster-provision/k8s/check-cluster-up.sh

```diff
@@ -12,21 +12,31 @@ function cleanup {
   make cluster-down
 }
 
-# check cluster-up
+# repetition cluster-up
 (
   ksh="./cluster-up/kubectl.sh"
   cd "$DIR" && cd ../..
   export KUBEVIRTCI_PROVISION_CHECK=1
   export KUBEVIRT_PROVIDER="k8s-${provision_dir}"
   export KUBEVIRT_NUM_NODES=2
   export KUBEVIRT_NUM_SECONDARY_NICS=2
+  if [[ ! "${provision_dir}" =~ ^1\.1[456]$ ]]; then
+    export KUBEVIRT_WITH_CNAO=true
```

Comment:
it means that sonobuoy would check with CNAO upon provision,
but usually people use the provider without CNAO installed, in the rare case that
with CNAO there isnt failures with sonobuoy and without it there is, we wont see it upon provision,
if its interesting..